### PR TITLE
Biodiv data exploration Tuto

### PR DESCRIPTION
**Pull request by yvanlebras**

### CODE REVIEW

File: topics/ecology/tutorials/Biodiversity data exploration/tutorial.md [hunk 1]
```diff
@@ -40,255 +46,231 @@ This tutorial will guide you on the exploration of biodiversity data having taxo
...
 First step is to download biodiversity data on your Galaxy history. Here we will use a "classical" (containing taxonomic, spatial and temporal informations) biodiversity dataset from the well known ["Reef life survey" initiative](https://reeflifesurvey.com/).
 
 ## Get data
 
 > ### {% icon hands_on %} Hands-on: Data upload
 >
-> 1. Create a new history for this tutorial
+> 1. Create a new history for this tutorial and give it a name (example: "RLS for biodiversity data exploration tutorial")
+>    for you to find it again later if needed.
 > 2. Import the files from [Zenodo](https://zenodo.org/record/5930763/files/Reel_life_survey_fish_sample.tabular?download=1)
```
Comment (ColineRoyaux) [inline topics/ecology/tutorials/Biodiversity data exploration/tutorial.md]: Change the link here to "https://zenodo.org/record/6107457/files/IMOS-National_Reef_Monitoring_Network_Sub-Facility-Global_reef_fish_abundance_and_biomass.csv?download=1"

File: topics/ecology/tutorials/Biodiversity data exploration/tutorial.md [hunk 2]
```diff
@@ -40,255 +46,231 @@ This tutorial will guide you on the exploration of biodiversity data having taxo
...
->    - {% icon param-file %} *"Input table"*: `output` (Input dataset)
->    - *"Select column containing latitudes in decimal degrees"*: `c9`
->    - *"Select column containing longitudes in decimal degrees"*: `c10`
+> 1. Use {% tool [Cut columns from a table (cut)](toolshed.g2.bx.psu.edu/repos/bgruening/text_processing/tp_cut_tool/1.1.0) %}
+>    with following parameters :
+>    - {% icon param-files %} *"File to cut"*: Convert CSV to tabular data file
+>    - {% icon param-select %} *"Operation"*: `Keep`
+>    - {% icon param-select %} *"Delimited by"*: `Tab`
+>    - {% icon param-select %} *"Cut by"*: `fields`
+>      - {% icon param-select %} *"List of Fields"*: `Column: 8` `Column: 10` `Column: 11` `Column: 12` `Column: 25` `Column: 28`
```
Comment (ColineRoyaux) [inline topics/ecology/tutorials/Biodiversity data exploration/tutorial.md:93]: Add the real header names of the columns and justify that we just keep the columns we need for the tutorial but it doesn't mean the other columns aren't interesting

File: topics/ecology/tutorials/Biodiversity data exploration/tutorial.md [hunk 3]
```diff
@@ -40,255 +46,231 @@ This tutorial will guide you on the exploration of biodiversity data having taxo
...
->    > ### {% icon comment %} Comment
->    >
->    > A comment about the tool or something else. This box can also be in the main text
->    {: .comment}
+> 1. {% tool [Spatial coordinates anonymization](toolshed.g2.bx.psu.edu/repos/ecology/tool_anonymization/tool_anonymization/0.0.0) %} with the following parameters:
+>    - {% icon param-file %} *"Input table"*: Column Regex Find and Replace data file
+>    - {% icon param-select %} *"First line is a header line"*: `Yes`
+>    - {% icon param-select %} *"Select column containing latitudes in decimal degrees"*: `c2`
+>    - {% icon param-select %} *"Select column containing longitudes in decimal degrees"*: `c3`
 >
```
Comment (ColineRoyaux) [inline topics/ecology/tutorials/Biodiversity data exploration/tutorial.md:119]: Add a warning => decimal degrees coordinates only

**Comment by hexylena** [comment]
@yvanlebras you discovered a nice bug, we couldn't handle spaces in filenames. I've fixed the scripts to help with this, but, I would still recommend you rename the folder to `biodiversity-data-exploration` so it'll look nicer in URLs as well.

also seems Marie59 is undefined

**Comment by hexylena** [comment]
I think i've caught the rest of the linting issues

File: topics/ecology/tutorials/Biodiversity data exploration/tutorial.md [hunk 4]
```diff
@@ -285,3 +266,22 @@ Final absence correlation plot:
 
 Here, you just explored your biodiversity dataframe properly and you know a lot more about your data. You can now peacefully make your statiscal analyses as most of the red flags you can get have been revealed by this toolsuite ! Enjoy !
 
+# Want to spatially anoymize your data?
+
+A first step of this tutorial will show you how you can simply apply spatial coordinates anonymization if you want to share data without spatial context, particularly of interest if you want to share endangered species oriented data.
+
+# Bonus!
+## Spatial coordinates anonymization
```
Comment (hexylena) [inline topics/ecology/tutorials/Biodiversity data exploration/tutorial.md:278]:
```suggestion
## Bonus! Spatial coordinates anonymization
```

Comment (hexylena) [reply topics/ecology/tutorials/Biodiversity data exploration/tutorial.md:278]: (just to try and keep the heading levels hierarchical.)

Comment (yvanlebras) [reply topics/ecology/tutorials/Biodiversity data exploration/tutorial.md]: THANK YOU Helena! Hope all is ok on your side!!!

**Review by hexylena — APPROVED** [review]

**Comment by hexylena** [comment]
Great work @yvanlebras, this is fantastic :)

**Comment by hexylena** [comment]
I'll merge this shortly

**Review by hexylena — APPROVED** [review]

**Review by hexylena — APPROVED** [review]

**Pull request merged**